ionic-template/slice.js

```diff
@@ -0,0 +1,39 @@
+import { createSlice } from '@reduxjs/toolkit'
+// import { selectorForSlice } from 'lp-redux-utils'
+// import * as apiActions from 'api-actions'
+// import { handleSuccess } from 'lp-redux-api'
+
+export const name = '%subsection%'
+const initialState = {}
+
+const slice = createSlice({
+  name,
+  initialState,
+  reducers: {
+    // clear%SubSection% () {
+    //   state.%subSection% = null
+    // }
+  },
+  extraReducers: (/* builder */) => {
+    // builder.addCase(
+    //   apiActions.fetch%SubSections%.toString(),
+    //   handleSuccess((state, action) => {
```

Comment:
I always liked using the `setOnSuccess` type functions. I think that should still work with RTK, right?
For reference:
```
function setOnSuccess (path, transform=getDataFromAction) {
  return handleSuccess((state, action) => set(path, transform(action, state), state))
}
```

I understand the use of Immer by RTK. Is there a reason/benefit to using direct assignment, as shown in the commented code below vs. Lodash `set`, as we have always done?

Reply:
`setOnSuccess` would still work!. I decided against using it because to me it did not introduce the shift in mindset of being able to make multiple, immutable changes through "mutation".

I like how it simplifies things when multiple properties need to get updated. Instead of
```js
handleSuccess((state, action) => {
  return compose(setState('todo', null), setState('todos', null))(state)
})
// or
handleSuccess((state, action) => {
  return {
    ...state,
   todos: null,
   todo: null,
  }
})
```
you can just do
```js
handleSuccess((state, action) => {
  state.todo = null
  state.todos = null
})
```
Granted, this isn't a huge win in terms of DX 🤷‍♂️